ios-plugin/pom.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.ecocode</groupId>
+        <artifactId>ecocode-mobile-parent</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>ios-plugin</artifactId>
+    <!--    <version>0.1.0-SNAPSHOT</version>-->
+    <packaging>pom</packaging>
+
+    <name>ecoCode iOS plugin</name>
+    <description>Help the earth, adopt this green plugin for your applications</description>
+    <url>https://github.com/cnumr/ecoCode</url>
+    <inceptionYear>2022</inceptionYear>
+
+    <modules>
```

Comment:
Hi @dedece35,
I renamed the modules but not the folders: here maven is expecting the folder path of the modules (not the module artifact name, otherwise it cannot find them).

I could renamed module folders as well with ecocode prefix, but it is a bit redundant and less readable in the source tree.

What do you think ?

Reply:
you're right. no problem to keep like this.